java/spacewalk-java.spec

```
@@ -96,7 +96,7 @@ BuildRequires:  dom4j
 BuildRequires:  dwr >= 3
 BuildRequires:  glassfish-jaxb-runtime
 BuildRequires:  glassfish-jaxb-txw2
-BuildRequires:  google-gson >= 2.2.4
+BuildRequires:  google-gson < 2.10.0
```

What about something like:
```suggestion
BuildRequires:  (google-gson >= 2.2.4 with google-gson < 2.10.0)
```

---

Let me try your suggestion. I tried getting both requirement statements to work but didn't succeed.